docs/miniCore.md

```diff
@@ -0,0 +1,194 @@
+# A Minimal Core and Clearly Defined Extension Points
+
+This proposal aims to define a minimal core and sensible extension points.
+
+
+## Consensus on Annotation Syntax
```

Comment:
Different, but not necessarily better. I'm reluctant to go into such detailed editing as long as the broad lines are not discussed.

Reply:
"Consensus" suggests that everyone has already agreed on this approach. I think that's a significantly premature label, and could easily cause the remainder of the document to be rejected on principle. I don't think that's what you want, so I suggest again that you change this heading.

Reply:
I wasn't aware that people are so picky about wording in a discussion document. The idea was to make discussions easier to follow by moving them from the mailinglist into the Github wiki. If we now start to treat them like official WG documents and discuss finegrained editorial changes, the discussion becomes equally hard to follow, and unproductive w.r.t. to the discussions we actually need to have. 
That said: would you rephrase your suggested resolution into "Annotation Syntax as a Common Ground"?

Reply:
People *in* the WG will interpret things differently than people *outside* the WG, and this document is visible to *all*. For this simple reason, we should be careful not to write things (like titles including "consensus" without "possible" or other qualifier) that imply support/acceptance by the entire WG until such time as that implication is accurate. Your suggested rephrasing is similarly problematic — it presents a *possibility* as if it had already been decided. Perhaps "A New Annotation Syntax as a Possible Common Ground" would satisfy you (as it would me)?

Reply:
The annotation syntax is not new.

Reply:
Changed suggestion to "The Annotation Syntax as a Possible Common Ground".